lib/libspl/include/os/freebsd/sys/byteorder.h

```diff
@@ -37,93 +37,15 @@
  * contributors.
  */
 
+#include <sys/endian.h>
```

Comment:
Typically includes go below the header guard, is there a reason for deviating here?